### PR TITLE
2282 Add gitlab actions for pushing Java containers to ECR

### PR DESCRIPTION
**Pull request by aberke220**
- credit-application-service
- accounts-service
- transactions-service

### CODE REVIEW

File: .github/workflows/ci-accounts-service.yml [hunk 1]
```diff
@@ -35,3 +35,51 @@ jobs:
     #  run: mvn deploy --file domain-service/credit-application/pom.xml -DskipTests
     #  env:
     #    GITHUB_TOKEN: ${{ github.token }}
+
+  pushToEcr:
```
Comment (theqajedi) [inline .github/workflows/ci-accounts-service.yml:39]: how do you trigger this section?

Comment (theqajedi) [reply .github/workflows/ci-accounts-service.yml:39]: ah if its a master branch.

**Comment by theqajedi** [comment]
Although we are pushing image on every merge to master, it will never be used until manually updated in fargate.
Or we can deploy it to a staging env (which we don't have ATM :D ).
So I would suggest having a rotation policy in the ECR repo (if not already setup) so that we won't have too many images just lying there. :)

**Review by theqajedi — APPROVED** [review]

**Comment by aberke220** [comment]
> Although we are pushing image on every merge to master, it will never be used until manually updated in fargate.
> Or we can deploy it to a staging env (which we don't have ATM :D ).
> So I would suggest having a rotation policy in the ECR repo (if not already setup) so that we won't have too many images just lying there. :)

@theqajedi It makes sense, I have added a follow-up ticket AB#2342 for cleaning-up old images and AB#2343 for rolling out new versions in Fargate

**Pull request merged**